Makefile.inc

```diff
@@ -45,9 +45,20 @@ endif
 
 # Shipyard provided targets
 
-cleanup:
+# clean cleans everything (running clusters, generated files ...)
+clean: clean-clusters clean-generated
+
+# clean-generated removes files we generated
+clean-generated:
+	git clean -X -f
+
+# clean-clusters removes running clusters
+clean-clusters:
 	$(SCRIPTS_DIR)/cleanup.sh
 
+# cleanup is the historical clean-clusters
+cleanup: clean-clusters
```

Comment:
I propose we just remove this. We could also have it just print a message that it's deprecated and then remove later but I think that would be overkill.

Reply:
Done.

Reply:
It turns out other projects need this, so it’s still in for now, but I’ll remove it ASAP.